src/controller/c/supervisor.c

```diff
@@ -89,21 +89,26 @@ static WbFieldRequest *field_requests_garbage_list = NULL;
 static WbFieldRequest *sent_field_get_request = NULL;
 static bool is_field_immediate_message = false;
 
+typedef struct WbNodeWbContactPointListStructPrivate {
+  int n;
+  WbContactPoint *points;
+  double timestamp;  // TODO: Delete with `wb_supervisor_node_get_contact_point`
```

Comment:
Can't we address this TODO?

Reply:
We can delete the `wb_supervisor_node_get_contact_point` functions, but we don't want that :)